plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEnginePublishOperation.java

```diff
@@ -51,18 +51,18 @@ protected IPath getModuleDeployDirectory(IModule module) {
   }
 
   @Override
-  protected void setModulePublishState(IModule[] module, int publishState) {
-    server.setModulePublishState2(module, publishState);
+  protected void setModulePublishState(IModule[] modules, int publishState) {
+    server.setModulePublishState2(modules, publishState);
```

Comment:
In WTP, the array represents a module path to the module.  For example, module[0] = a web module, module[1] = utility module; the last module is the actual module.  So these should remain the singular `module`.